parquet-cli/src/main/java/org/apache/parquet/cli/commands/ShowSizeStatisticsCommand.java

```diff
@@ -60,9 +72,13 @@ public int run() throws IOException {
 
       console.info("\nFile path: {}", source);
 
-      List<BlockMetaData> rowGroups = footer.getBlocks();
-      for (int index = 0, n = rowGroups.size(); index < n; index++) {
-        printRowGroupSizeStats(console, index, rowGroups.get(index), schema);
+      List<BlockMetaData> blocks = footer.getBlocks();
+      Set<Integer> allowedRowGroups = rowGroups == null ? null : new HashSet<>(rowGroups);
```

Comment:
nit: If you would create an empty Set instead of initializing with a null, you wouldn't have to check for null in the loop.